src/connectors/injected/index.ts

```diff
@@ -144,7 +144,7 @@ export class InjectedConnector extends Connector {
       params: { silent_mode: true },
     })
 
-    return new Account(provider, accounts[0], "")
+    return new WalletAccount(provider, this._wallet, undefined, accounts[0])
```

Comment:
I guess this is not a breaking change, just an addition to everything `Account` has? (Since `WalletAccount extends Account`, probably?